packages/nextjs-mf/src/plugins/NextFederationPlugin/apply-client-plugins.ts

```diff
@@ -33,8 +33,10 @@ export function applyClientPlugins(
 ): void {
   const { webpack } = compiler;
   const { remotes, name } = options;
-  //@ts-ignore
-  compiler.options.output.publicPath = 'auto';
+
+  if (compiler.options.output.publicPath === '/_next/') {
+    compiler.options.output.publicPath = 'auto';
```

Comment:
what are you setting public path to? absolute url or relative?

Reply:
On my implementation you mean?
I was setting absolute, the complete url.

Reply:
Okay.